### ISSUE TITLE
Indentation not checked on companion object

### ISSUE THREAD

**Issue by JLLeitschuh** [shyiko/ktlint#324]
Example (which I think should fail):

```kotlin
class Test {
        companion object {
    }
}
```

Ktlint version: `0.28.0`

**Comment by shyiko** [comment]
Moved to #338.
